Userland/Libraries/LibGUI/Widget.h

```diff
@@ -71,28 +71,13 @@ enum class AllowCallback {
     Yes
 };
 
-template<typename T>
-concept HasFalliblesInitializer = requires(T object) {
-    {
-        object.initialize_fallibles()
-    } -> SameAs<ErrorOr<void>>;
+auto initialize = [](auto& object) -> ErrorOr<void> {
+    if constexpr (requires { { object.initialize() } -> SameAs<ErrorOr<void>>; })
+        return object.initialize();
+    else
+        return {};
 };
```

Comment:
This deletes what the first commit added, not sure about that. But also, this just puts a global variable in a header file, which the CI compilers are rightly yelling at you about.  If you want it as a template function rather than an artifact of the code generator, it should look more like this:

```
template<DerivedFrom<::GUI::Widget> T>
ErrorOr<void> initialize(T& object) {
    if constexpr (requires { { object.initialize() } -> SameAs<ErrorOr<void>>; })
        return object.initialize();
    else
        return {};
}
```

Reply:
I was thinking that that initialize lambda would just be created on the stack in the generated code when it wants to init the class. Similar to how it works for JS::Cells that are created by the JS::Heap